curvefs/proto/metaserver.proto

```diff
@@ -242,6 +242,8 @@ message CreateInodeRequest {
     required uint64 parent = 10;
     optional uint64 rdev = 11;
     optional string symlink = 12;   // TYPE_SYM_LINK only
+    optional uint64 createsec = 13;
```

Comment:
`CreateRootInodeRequest` also has the same problem.

Reply:
> `CreateRootInodeRequest` also has the same problem.

fix